pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -442,6 +449,16 @@ func (c *InstallerController) manageInstallationPods(ctx context.Context, operat
 		return true, requeueAfter, nil
 	}
 
+	// Check if revision should be installed based on if the quorum is about to be violated
+	if c.shouldRevisionInstall != nil {
+		quorumSafe, err := c.shouldRevisionInstall()
+		if !quorumSafe {
+			return true, 30 * time.Second, err
```

Comment:
- when quorumSafe failed, the err is not nil, then  return `30 * time.Second`makes no sense, what about `return true, 0, err`
- please see https://github.com/openshift/library-go/blob/aed018c215a122871be1768155cf9f3e658278fc/pkg/operator/staticpod/controller/installer/installer_controller.go#L1070-L1074